### PR TITLE
Sunset Django People

### PR DESCRIPTION
**Pull request by carltongibson**
Add page to redirect people.djangoproject.com to, and allow sun-setting that. 

### Adjusts community header, which was using stats from django people. 

Before: 

Screenshot 2020-09-08 at 12 18 00

Looks like there's a bug in that — presumably packages API is down... 

After: 

Screenshot 2020-09-08 at 11 56 32

From Django 3.2 we can use the `depth` to `timesince` to just say `15 years`, but I don't see the value in writing code for that till we get there. 

### Adds a flatpage to redirect to. 

* https://www.djangoproject.com/community/django-people/ 
* point on to `/community/` 
* Screenshot added to use for that. (Not live so HTML won't be right. Not sure if I can use `{% static %}` in a flatpage...🤔)

//cc @frankwiles @wsvincent just FYI, but any comments welcome. 🙂

### CODE REVIEW

File: aggregator/context_processors.py [hunk 1]
```diff
@@ -1,37 +1,16 @@
...
-        packages_data = fetch(PACKAGES_STATS_URL)
-        if 'meta' in packages_data:
-            stats.update({'packages': packages_data['meta']['total_count']})
-
-        stats = {'community_stats': stats}
-
-        cache.add(STATS_CACHE_KEY, stats, 60 * 60 * 12)  # for half a day
-
-    return stats
+    # Django 3.2 introduces depth kwarg. Set timesince(..., depth=1) then.
```
Comment (carltongibson) [inline aggregator/context_processors.py:14]: Could do an `if django.VERSION...` but...

File: djangoproject/templates/base_community.html [hunk 2]
```diff
@@ -6,11 +6,8 @@
 
 {% block header %}
   <h1 class="visuallyhidden">Community</h1>
-  <p><em>Django</em> Community{% if community_stats %}
-  {% if community_stats.people %} <em>{{ community_stats.people }}</em> people,{% endif %}
-  {% if community_stats.countries %} <em>{{ community_stats.countries }}</em> countries,{% endif %}
-  {% if community_stats.packages %} <em>{{ community_stats.packages }}</em> packages and projects.{% endif %}
-  {% endif %}</p>
+  <p>Building the <em>Django</em> Community{% if community_stats %}
+  {% if community_stats.age %} for <em>{{ community_stats.age }}</em>{% endif %}{% endif %}. Come join us!</p>
```
Comment (felixxm) [inline djangoproject/templates/base_community.html:10]: I would totally remove stats:
```suggestion
  <p><em>Django</em> Community. Come join us!</p>
```

**Review by felixxm — COMMENTED** [review]
@carltongibson Thanks :+1:

File: djangoproject/templates/base_community.html (hunk 2 above)
Comment (carltongibson) [reply djangoproject/templates/base_community.html:10]: I quite liked the 15 years thing... — It's Authority building, like the current stats. It says _This is a big project_ you can trust it_. (like those client logos people put on their web sites.) 

If you don't buy that we can drop it. 😀

Comment (felixxm) [reply djangoproject/templates/base_community.html:10]: `timesince()` is translated so it is not consistent, we should unlocalize it

obraz

Comment (carltongibson) [reply djangoproject/templates/base_community.html:10]: OK, I'll write a test for that.

**Comment by frankwiles** [comment]
LGTM!

File: djangoproject/templates/base_community.html (hunk 2 above)
Comment (carltongibson) [reply djangoproject/templates/base_community.html]: OK, updated.

**Pull request merged**